main.go

```diff
@@ -354,12 +335,25 @@ func main() {
 	interrupted := make(chan os.Signal, 2)
 	signal.Notify(interrupted, syscall.SIGINT)
 
+	var met metrics.Metrics
+
+	switch strings.ToLower(*metricsServerBackend) {
+	case ServerBackendPrometheus:
+		met = metrics.NewPrometheus()
```

Comment:
call we have a more descriptive name? met is not obvious what this is.
How about metricsBackend?